index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Code Partner Challenge</title>
+    <link rel="stylesheet" href="style1.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+
+    <main id="main">
+      <div id="title">
+        <div>
+          <h1>Dr. Norman Balough</h1>
+          <p>The man who saved a billion lives</p>
+        </div>
+      </div>
+      <div id="img-div">
+         
+          <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="" id="image">
```

Comment:
The Alt attribute is a core foundation of semantic layout.
Please add a descriptive alt atribute so that the screenreader can read it.